structuredcollaboration/models.py

```diff
@@ -153,3 +162,46 @@ def set_policy(self, p):
     def __unicode__(self):
         return u'%s %r <%s %s> [%s]' % (self.title, self.pk, self.content_type,
                                         self.object_pk, self.slug)
+
+
+CollaborationPolicyRecord.objects.register_policy(
+    CollaborationPolicy, 'forbidden', 'Forbidden to everyone')
+
```

Comment:
Agreed. I'll take a look at AppConfig.ready and think through an alternate approach.